### ISSUE TITLE
Improvements to polar projections

### ISSUE THREAD

**Issue by jlstevens** [ioam/holoviews#1531]
Just a few observations regarding polar projections:

* The bokeh backend doesn't seem to support this option.
* The polar option seems to assume the kdim ranges from 0 to 2pi. It would be good if it either wrapped the data around no matter the kdim range or required the kdim range to be specified.

**Comment by jbednar** [comment]
> It would be good if it either wrapped the data around no matter the kdim range

I agree -- having things wrap around conveniently regardless of the actual value is one of the reasons to use a polar plot in the first place; it should have a soft range of 0,2pi but  no hard limits (though I can't remember whether Dimension has soft ranges).
